units/crons.go

```diff
@@ -320,6 +322,34 @@ func PopulateLastContainerFinishTimeJobs() amboy.QueueOperation {
 	}
 }
 
+func PopulateParentDecommissionJobs() amboy.QueueOperation {
+	return func(queue amboy.Queue) error {
+		catcher := grip.NewBasicCatcher()
+		ts := util.RoundPartOfHour(1).Format(tsFormat)
+
+		// Find all active distros supporting containers
+		query := db.Query(bson.M{
+			distro.DisabledKey: bson.M{
+				"$exists": false,
+			},
+			distro.MaxContainersKey: bson.M{
```

Comment:
are we still using this in until we take `MaxContainersKey` out of distro and refactoring later?

Reply:
Yes, we're still using MaxContainersKey for now -- will change this as part of the next ticket I'm working on.